pkg/brokerapi/openservicebroker/open_service_broker_client.go

```diff
@@ -19,24 +19,62 @@ package openservicebroker
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"net/http"
 	"time"
 
 	"github.com/golang/glog"
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 	"github.com/kubernetes-incubator/service-catalog/pkg/brokerapi"
-	"github.com/kubernetes-incubator/service-catalog/pkg/controller/util"
+	"github.com/kubernetes-incubator/service-catalog/pkg/util"
 )
 
 const (
 	catalogFormatString         = "%s/v2/catalog"
 	serviceInstanceFormatString = "%s/v2/service_instances/%s"
+	pollingFormatString         = "%s/v2/service_instances/%s/last_operation"
 	bindingFormatString         = "%s/v2/service_instances/%s/service_bindings/%s"
 
-	httpTimeoutSeconds = 15
+	httpTimeoutSeconds     = 15
+	pollingIntervalSeconds = 1
+	pollingAmountLimit     = 6
```

Comment:
Yeah, sure! I originally had the `pollingIntervalSeconds` quite a bit larger, but it made the tests run slowly, so I changed that but did not adjust the amount limit.